internal/controller/pipelinerollout_controller_test.go

```diff
@@ -288,3 +290,49 @@ var _ = Describe("PipelineRollout Controller", Ordered, func() {
 		})
 	})
 })
+
+func TestPipeLinLabels(t *testing.T) {
```

Comment:
thank you for adding this :)  although, going forward, don't worry about adding a test for pretty simple logic like this